test_unit/test_message.cpp

```diff
@@ -189,21 +189,26 @@ TEST(Message, localtime_formatted) {
 #endif // timezone 
 
 
+#if (defined(CHANGE_G3LOG_DEBUG_TO_DBUG))
+#define DEBUG_NAME DBUG
+#else
```

Comment:
This would be better to keep in the generated options file. Currently you are repeating it in multiple places which should not be needed as long as they are including the generated header file